src/main/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/impl/HashPartitioningItemMapper.java

```diff
@@ -53,7 +54,7 @@
         Map<String, AttributeValue> physicalItem = new HashMap<>(virtualItem);
 
         // copy table primary key fields
-        addPhysicalKeys(virtualItem, physicalItem, virtualTable.getPrimaryKey(), physicalTable.getPrimaryKey(), true);
+        addPhysicalKeys(virtualItem, physicalItem, virtualTable.getPrimaryKey(), physicalTable.getPrimaryKey(), false);
```

Comment:
This is because I want to be able to use this to convert the field-value map represented in an update expression. For put requests, now without this validation, if the item is missing a table PK field, then the mapped item will be missing the physical HK and RK fields, and when we do the put on the parent dynamo client, it will fail. (Adding a test for this now.)